api/v1alpha1/flowcollector_types.go

```diff
@@ -45,6 +45,9 @@ type FlowCollectorSpec struct {
 
 	// CNO contains settings related to the cluster network operator
 	CNO ClusterNetworkOperator `json:"cno,omitempty"`
+
+	// Operators contains compatible operators names to install, comma separated E.g. "loki-operator,strimzi-kafka-operator,grafana-operator"
+	Operators string `json:"op,omitempty"`
```

Comment:
@jpinsonneau can you add also Prometheus Operator so we will have the destination for the metrics?

@jotak @stleerh FYI ^^^

Reply:
sure, I'll take a look :+1:

Reply:
One thing about loki, it may feel weird to the user to enable installing the operator but still having to populate the loki URL in a different section. BTW how does it work, if the user installs the loki operator and set a URL that doesn't match it, are we going to ignore the URL set and use the known one instead? Should we document that the URL field is ignored when the operator is installed as a dependency?

So what do you think, if the loki section looks like:

```yaml
  loki:
    installOperator: true
    # url: '' (ignored)
    # querierUrl: '' (ignored)
    batchWait: 1s
```

And then, we could introduce new similar section for grafana / kafka / prometheus with a field `installOperator` true or false.